src/components/dashboard/FaceVerification/sdk/FaceTecSDK.native.js

```diff
@@ -11,8 +11,7 @@ import { MAX_RETRIES_ALLOWED } from './FaceTecSDK.constants'
 
 export { FaceTecSDKStatus, FaceTecSessionStatus } from '@gooddollar/react-native-facetec'
 
-// sdk class
-export const FaceTecSDK = new class {
```

Comment:
this wasn't liked by the upgraded eslint had to declare class then instantiate it separately
not sure why exactly because in the .web version  were no issues with the same statement (anonymous class instantiation)